web_favicon/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
In python3 `# -*- coding: utf-8 -*-` it should be removed.

Same for the others python file in this pull request.